pkg/api/handlers/utils/handler.go

```diff
@@ -59,3 +61,19 @@ func FilterMapToString(filters map[string][]string) (string, error) {
 	}
 	return string(f), nil
 }
+
+func getVar(r *http.Request, k string) string {
+	return mux.Vars(r)[k]
+}
+
+// GetName extracts the name from the mux and does unescape on
+// it.  Any error from the unescape is logged as Info because
+// the error is a user input error
+func GetName(r *http.Request) string {
+	val := getVar(r, "name")
+	name, err := url.PathUnescape(val)
```

Comment:
1) I would move the PathUnescape() into getVar() with the logging.  Also Wrapf() the error to provide some context on what is failing.
2) We agreed to return the value of the variable on errors not "" which this code does.